app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,39 @@
+class RelationshipsController < ApplicationController
+  before_action :logged_in_user, only: [:create, :destroy]
+  before_action :load_user, only: :create
+  before_action :load_relationship, only: :destroy
+
+  def create
+    current_user.follow @user
+    respond_to do |format|
+      format.html{redirect_to @user}
+      format.js
+    end
+  end
+
+  def destroy
+    current_user.unfollow @relationship.followed
+    respond_to do |format|follow
+      format.html{redirect_to @user}
```

Comment:
Object @user đâu mak ông redirect to thế kia. Ưa bị cắn hơm

Reply:
@@ ko ko. Tui biết tui sai ở đâu rồi. hì hì